pkgs/tools/audio/beets/plugins/filetote.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  beets,
+  beetsPackages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "beets-filetote";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "gtronset";
+    repo = "beets-filetote";
+    rev = "v${version}";
```

Comment:
Using `tag` instead of `rev` here prevents issues if a branch with the same name is created.